### ISSUE TITLE
myub.buffalo.edu redirects to myubbuffalo.edu

### ISSUE THREAD

**Issue by XVicarious** [EFForg/https-everywhere#3237]
I was wondering what was causing this issue for me to get to my school's portal.

Today I went through my plugins and disabled them one by one and after I disabled HTTPS Everywhere, it went from taking forever to resolve (and eventually failing) to going there right away.

```
URL to access:
myub.buffalo.edu

URL it tries to bring me to:
myubbuffalo.edu
```

**Comment by fuglede** [comment]
Thanks for the report.

**Comment by loveithateit** [comment]
Fixed in cf4b2bad03d753d5911c4239c4691daaf88eb7ca
